pytition/cli_pytition.py

```diff
@@ -84,6 +91,27 @@ def main():
 	elif args.action == "gen_orga":
 		org = Organization.objects.create(name=args.orga)
 		org.save()
+	elif args.action == "generate_signatures":
+		if args.petition_id is not None:
+			petition = Petition.objects.get(id=args.petition_id)
+		elif args.petition_title is not None:
+			try:
+				petition = Petition.objects.get(title=args.petition_title)
+			except django.core.exceptions.MultipleObjectsReturned:
+				print("multitple petition match this title, choose it by id")
+				sys.exit(1)
+		else:
+			print("You must either specify --petition-id or --petition-title")
+			sys.exit(1)
+
+		for i in range(args.number):
+			signature = Signature.objects.create(
+				first_name=args.first_name,
+				last_name=args.last_name,
+				email=args.email,
+				petition=petition
+			)
+			signature.save()
```

Comment:
No need to .save() if objects.create() is used.
(Yes, the other parts of the program is also doing it wrongly ^^)